### PR TITLE
Add versor v0.0.1 to BCR

### PR DESCRIPTION
**Pull request by udaya2899**
versor doesn't have a release on github. I can only use the tag URL from github.

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (versor) have been updated in this PR. Please review the changes.

**Comment by udaya2899** [comment]
Requesting maintainers to add the label to skip the unstable source URL validation as github.com/wolftype/versor doesn't have a release

File: modules/versor/0.0.1/MODULE.bazel [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module(
+    name = "versor",
+    version = "0.0.1",
```
Comment (mering) [inline modules/versor/0.0.1/MODULE.bazel:3]: Prefer to use `0.0.0-YYYYMMDD-SHORTSHA` as we do for example in `boringssl`.

File: modules/versor/0.0.1/patches/add_build_file.patch [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
++        "include",
++        "include/vsr",
++        "include/vsr/detail",
++    ],
++)
++
++build_test(
++    name = "versor_build_test",
++    targets = [":vsr"],
++)
```
Comment (mering) [inline modules/versor/0.0.1/patches/add_build_file.patch:47]: I think we do not need to add a special test just for building. Building is tested via `presubmit.yml` already. We could probably get rid of the `bazel_skylib` dependency then as well.

File: modules/versor/0.0.1/patches/add_build_file.patch [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
++
++license(
++    name = "license",
++    package_name = "versor",
++)
++
++licenses(["notice"])
++
++exports_files(["copyrights/COPYRIGHT"])
```
Comment (mering) [inline modules/versor/0.0.1/patches/add_build_file.patch:22]: Is this one of the default licenses? If so, you could add the appropriate license kind and also reference the `COPYRIGHT` file in the license metadata.

File: modules/versor/0.0.1/patches/module_dot_bazel.patch [hunk 4]
```diff
@@ -0,0 +1,11 @@
+--- a/MODULE.bazel
++++ a/MODULE.bazel
+@@ -0,0 +1,8 @@
++module(
++    name = "versor",
++    version = "0.0.1",
++    compatibility_level = 0,
```
Comment (mering) [inline modules/versor/0.0.1/patches/module_dot_bazel.patch:7]: Nit: One doesn't need to set this explicitly.

**Review by meteorcloudy — APPROVED** [review]

**Pull request merged**